blockchains/eth/eth_worker.js

```diff
@@ -29,6 +30,32 @@ class ETHWorker extends BaseWorker {
     this.withdrawalsDecoder = new WithdrawalsDecoder(this.web3, this.web3Wrapper);
   }
 
+  async ethClientRequestWithRetry(...params) {
+    let retries = 0;
+    let retryIntervalMs = 0;
+    while (retries < constants.MAX_RETRIES) {
+      try {
+        const response = await this.ethClient.request(...params);
+        if (response.error || response.result === null) {
+          retries++;
+          retryIntervalMs += (2000 * retries);
+          logger.error(`${params[0]} failed. Reason: ${response.error}. Retrying for ${retries} time`);
```

Comment:
Is it really the `params[0]` that failed? That's one request out of the batch, maybe just summarize that a batch of size `params.length()` has failed.

Reply:
param[0] is the method name. 'eth_getLogs' for example